okapi-core/src/main/java/org/folio/okapi/managers/TenantManager.java

```diff
@@ -572,11 +573,11 @@ private void fireTimer(Tenant tenant, ModuleDescriptor md, RoutingEntry re, Stri
           "Bad _tenantPermissions interface in module " + permsModule.getId()
               + ". No path to POST to"));
     }
-    pc.debug("tenantPerms: " + permsModule.getId() + " and " + permPath);
+    logger.debug("tenantPerms: {} and {}", (Supplier<?>) () -> permsModule.getId(), permPath);
     return proxyService.callSystemInterface(tenant, permInst, pljson, pc).compose(cres -> {
       pc.passOkapiTraceHeaders(cres);
-      pc.debug("tenantPerms request to " + permsModule.getName()
-          + " succeeded for module " + moduleTo + " and tenant " + tenant.getId());
+      logger.debug("tenantPerms request to {} succeeded for module {} and tenant {}",
+          () -> permsModule.getId(), () -> moduleTo, () -> tenant.getId());
```

Comment:
Would this work `logger.debug("tenantPerms request to {} succeeded for module {} and tenant {}", permsModule.getId(), moduleTo, tenant.getId());`?

Reply:
yes